packages/gamut/src/Dialog/index.tsx

```diff
@@ -11,6 +11,10 @@ import { Text } from '../Typography';
 
 const ShroudedOverlay = styled(Overlay)(system.css({ bg: 'shadow-opaque' }));
 
+type DialogButtonProps = Pick<
+  ComponentProps<typeof FillButton>,
+  'href' | 'onClick' | 'children'
+>;
```

Comment:
Nit - Room to breathe
```suggestion
type DialogButtonProps = Pick<
  ComponentProps<typeof FillButton>,
  'href' | 'onClick' | 'children'
>;

```